### PR TITLE
Added swag for JabRef Hacktoberfest 2019

### LINKED ISSUE #364: Hacktoberfest 2019: JabRef
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : JabRef

Q. Where have you seen the announcement (URL)?  
A. : https://www.jabref.org/hacktoberfest/2019.html

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, stickers

Q. Have you received swag from this?  
A. : no

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Dunno

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by LeeVaughn**
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate pull request.
- [x] I've checked that this is linked to a new swag opportunity issue <!>

Implement swag opportunity: JabRef Hacktoberfest 2019

Closes #364
  
**Things to check:**

- [x] Date Added is today (the date of the PR)
- [x] Opportunity is in alphabetical order

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -143,6 +143,15 @@
     "dateAdded": "2019-08-20T19:20:42.000Z",
     "tags": ["stickers", "clothing"]
   },
+  {
+    "name": "JabRef",
+    "difficulty": "medium",
+    "description": "Get a pull request merged, receive a sticker. Get five pull requests merged, receive a T-shirt.",
+    "reference": "https://www.jabref.org/hacktoberfest/2019.html",
+    "image": "https://hacktoberfest.digitalocean.com/assets/logo-hf19-header-8245176fe235ab5d942c7580778a914110fa06a23c3d55bf40e2d061809d8785.svg",
```
Comment (Yash-Garg) [inline data.json]:
```suggestion
    "image": "https://hacktoberfest.digitalocean.com/assets/logo-hf19-header-8245176fe235ab5d942c7580778a914110fa06a23c3d55bf40e2d061809d8785.svg",
    kk
```

Comment (Yash-Garg) [inline data.json]:
```suggestion
    "image": "https://hacktoberfest.digitalocean.com/assets/HF19_social-744d976f227e4aff6866443abcede8c651b309ec9c7c9f7410f5944f8e1299b9.png",
```

File: data.json [hunk 2]
```diff
@@ -143,6 +143,15 @@
     "dateAdded": "2019-08-20T19:20:42.000Z",
     "tags": ["stickers", "clothing"]
   },
+  {
+    "name": "JabRef",
+    "difficulty": "medium",
+    "description": "Get a pull request merged, receive a sticker. Get five pull requests merged, receive a T-shirt.",
+    "reference": "https://www.jabref.org/hacktoberfest/2019.html",
+    "image": "https://hacktoberfest.digitalocean.com/assets/HF19_social-744d976f227e4aff6866443abcede8c651b309ec9c7c9f7410f5944f8e1299b9.png",
+    "dateAdded": "2019-10-5T23:07:46.000Z",
```
Comment (Yash-Garg) [inline data.json]:
```suggestion
    "dateAdded": "2019-10-05T23:07:46.000Z",
```

**Comment by swapagarwal** [comment]
Thanks @LeeVaughn! 👍

**Pull request merged**